README.md

```diff
@@ -258,15 +245,16 @@ ssh -o ProxyCommand="ssh -W%h:%p %r@<deployer_concourse_ip>" vcap@10.0.0.6
 
 ## Concourse credentials
 
-`./vagrant/deploy.sh` generates the concourse ATC password for the admin user,
-based on the AWS credentials, the environment name and the application name.
+By default, the environment setup script generates the concourse ATC password
+for the admin user, based on the AWS credentials, the environment name and the
+application name. If the `CONCOURSE_ATC_PASSWORD` environment variable is set,
+this will be used instead. These credentials are output by all of the pipeline
+deployment tasks.
 
 These credentials will also be used by the *Deployer Concourse*.
 
-If you are the owner of the environment with the original AWS credentials,
-run `TARGET_CONCOURSE=bootstrap ./concourse/scripts/environment.sh` to get them again.
-
-If not, it can be found in the `basic_auth_password` property of `concourse-manifest.yml` in the state bucket.
+If necessary, the concourse password can be found in the `basic_auth_password`
+property of `concourse-manifest.yml` in the state bucket.
```

Comment:
`aws s3 cp` can write to STDOUT and `val_from_yaml.rb` can read from STDIN, so we could avoid using a temporary file at all. Probably one for the Makefile though, related to this story: https://www.pivotaltracker.com/story/show/113160603